src/incubator/f5-bigip-ingress/README.md

```diff
@@ -1,12 +1,25 @@
-# alpha status
+# Helm Chart for using the F5 BIG-IP as an ingress
 
-This chart is in an alpha status. It can be used to deploy an ingress with on simple service. 
+This chart simplifies repeatable, versioned use of the F5 BIG-IP as an [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) within Kubernetes or OpenShift by using the [k8s-bigip-ctlr](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-ingress.html) as an ingress controller.
```

Comment:
"as an ingress" --> "to manage Ingress resources ..."

or, change the whole sentence to
```
"This chart simplifies repeatable, versioned use of the [F5 BIG-IP Controller as an Ingress Controller](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-k8s-ingress-ctlr.html) in Kubernetes or OpenShift. "
```
(link points to new doc coming out with grays release)

Reply:
Used suggested alternative, thanks. :)